bindings/matrix-sdk-crypto-js/src/sync_events.rs

```diff
@@ -9,7 +9,7 @@ use crate::{downcast, identifiers};
 #[wasm_bindgen]
 #[derive(Debug)]
 pub struct DeviceLists {
-    pub(crate) inner: ruma::api::client::sync::sync_events::v3::DeviceLists,
+    pub(crate) inner: ruma::api::client::sync::sync_events::DeviceLists,
```

Comment:
Why changing this?

Reply:
Because it had moved. In the final version it is still available in v3, too ... So we could leave that untouched (though I feel that using the non prefixed one is more accurate).

Reply:
The one inside `v3` is going to be removed, thanks for reminding me so I can just do it now 😄